psl-core/src/main/java/org/linqs/psl/config/Options.java

```diff
@@ -303,6 +265,13 @@
         null
     );
 
+    public static final Option WLA_GPP_USE_PROVIDED_WEIGHT = new Option(
+        "gpp.useProvidedWeight",
+        true,
+        "Whether the weight configuration in the user provided model file should be used as the initial "
+        + "sample point in GPP."
```

Comment:
Move the space to the beginning of this string (instead of the end of the previous line).